src/OpenLoco/S5/SawyerStream.cpp

```diff
@@ -0,0 +1,488 @@
+#include "SawyerStream.h"
+#include "../Utility/Numeric.hpp"
+#include <algorithm>
+#include <cassert>
+#include <cstring>
+#include <memory>
+#include <stdexcept>
+
+#ifdef _WIN32
+#ifndef NOMINMAX
+#define NOMINMAX
+#endif
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#endif
+
+using namespace OpenLoco;
```

Comment:
This whole file should probably be _in_ the OpenLoco namespace rather than using it.